trading_bot/ops.py

```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 
+global block
```

Comment:
Why are there 2 global variables initialized here with the same name?

Reply:
Because I need one inside the function and one outside. Pycharm is a little weird with this stuff. If you want tho, u can remove the one inside the function.